base/bitarray.jl

```diff
@@ -235,8 +235,11 @@ function fill!(B::BitArray, x)
     return B
 end
 
-falses(args...) = fill!(BitArray(args...), false)
-trues(args...) = fill!(BitArray(args...), true)
+for (f, v) in ((:falses, false), (:trues, true))
```

Comment:
> Honestly, I don't think the @eval machinery is worth it here. It makes the code much harder to read.

After making two of the cases just call the third instead of all of them calling `fill!`, I couldn't stop myself. But you're probably right.

>Also, do you know if both variants are currently tested?

No, but even if both of them get implicitly tested somewhere by just using them, it's probably a good idea to explicitly test them as part of the tests of this PR.